shard-operator-parent/shard-operator/src/main/kubernetes/kubernetes.yml

```diff
@@ -18,6 +18,19 @@ rules:
       - delete
       - patch
       - update
+  - apiGroups:
```

Comment:
This is already defined below - https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/shard-operator-parent/shard-operator/src/main/kubernetes/kubernetes.yml#L134-L148

Reply:
yes..I removed it